testcases/crypto/abfunc.c

```diff
@@ -1191,6 +1191,7 @@ int main(int argc, char **argv)
         }
     } else {
         rc = 0;
+        printf("%s only supported on the EP11 token.\n", argv[0]);
```

Comment:
Maybe better use testsuite_skip() so that it produces a correct SKIP message?

Reply:
well, I mean testcase_skip() ....

Reply:
Will do, good idea, the skip message is even more informative.
Thanks.